hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -2262,4 +2265,104 @@ boolean snapshotCleanupSwitch(final boolean on, final boolean synchronous)
    */
   boolean isSnapshotCleanupEnabled() throws IOException;
 
+  /**
+   * Creates a new RegionServer group with the given name
+   * @param groupName the name of the group
+   * @throws IOException if a remote or network exception occurs
+   */
+  void addRSGroup(String groupName) throws IOException;
```

Comment:
Do we still need 'RS'? Should this be addGroup or addRegionServerGroup?

Reply:
Hmm... Yeah, Group is too generic so s/RS/RegionGroup/ ? Big change I suppose.

Reply:
The IA.Public data structure is call RSGroupInfo so...

Reply:
Ok. So, your argument is that folks using current data structures and methods will have an easier time if the names of methods, etc., have same basic pattern in the new context.

Ok.